politeiawww/cmswww.go

```diff
@@ -367,6 +367,38 @@ func (p *politeiawww) handleCMSPolicy(w http.ResponseWriter, r *http.Request) {
 	util.RespondWithJSON(w, http.StatusOK, reply)
 }
 
+// handlePayInvoices handles the request to generate all of the payouts for any
+// currently approved invoice.
+func (p *politeiawww) handlePayInvoices(w http.ResponseWriter, r *http.Request) {
+	log.Tracef("handlePayInvoices")
+
+	// Get pay invoices command
+	var pi cms.PayInvoices
+	decoder := json.NewDecoder(r.Body)
```

Comment:
This is a GET request.  It doesn't have a body.  You would normally parse the query params here, but it doesn't have any so no need to.